### PR TITLE
coal golems

### PR DESCRIPTION
**Pull request by N3X15**
- coal golems

NOTE: coal golems

### CODE REVIEW

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 1]
```diff
@@ -1047,7 +1047,7 @@ var/global/list/damage_icon_parts = list()
 			//if(!old_tail_state) //only update if we didnt show our tail already
 
 				//overlays_standing[TAIL_LAYER] = image("icon" = 'icons/effects/species.dmi', "icon_state" = "[species.tail]_s")
-//				to_chat(src, "update: tail is different")
+//				to_chat(src, "update: tail is different")
```
Comment (N3X15) [inline code/modules/mob/living/carbon/human/update_icons.dm:1050]: QUIT MIXING YOUR GODDAMN LINE-ENDINGS JESUS CHRIST

File: code/modules/mob/living/carbon/slime/slime.dm [hunk 2]
```diff
@@ -995,16 +995,6 @@ mob/living/carbon/slime/var/temperature_resistance = T0C+75
...
-
-/obj/item/clothing/mask/gas/golem
-	name = "golem's face"
-	desc = "the imposing face of an adamantine golem"
-	icon_state = "golem"
-	item_state = "golem"
-	canremove = 0
-	siemens_coefficient = 0
-	unacidable = 1
-
```
Comment (N3X15) [inline code/modules/mob/living/carbon/slime/slime.dm:1007]: This was a duplicate definition and it set off my OCD

File: code/modules/spells/targeted/flash_to_coal.dm [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	for(var/mob/living/carbon/human/H in targets)
+		H.dna.mutantrace = "coalgolem"
+		H.real_name = text("Coal Golem ([rand(1, 1000)])")
+		H.equip_to_slot_or_del(new /obj/item/clothing/under/golem/coal(H), slot_w_uniform)
+		H.equip_to_slot_or_del(new /obj/item/clothing/suit/golem/coal(H), slot_wear_suit)
+		H.equip_to_slot_or_del(new /obj/item/clothing/shoes/golem/coal(H), slot_shoes)
+		H.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/golem/coal(H), slot_wear_mask)
+		H.equip_to_slot_or_del(new /obj/item/clothing/gloves/golem/coal(H), slot_gloves)
+		//G.equip_to_slot_or_del(new /obj/item/clothing/head/space/golem(H), slot_head)
+		to_chat(H, "You are a coal golem. You move slowly, but are highly resistant to heat and cold as well as blunt trauma. You are unable to wear clothes, but can still use most tools. Serve [user], and assist them in completing their goals at any cost.")
```
Comment (ghost) [inline code/modules/spells/targeted/flash_to_coal.dm]: Probably best you add them to the antag list as well just to save some hassle

File: code/modules/spells/targeted/flash_to_coal.dm [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+/obj/item/clothing/head/space/golem/coal
+	name = "coal head"
+	icon_state="golem_coal"
+
+/spell/targeted/flesh_to_coal
+	name = "Flesh to Coal"
+	desc = "This spell turns a single person into a coal golem slaved to the caster."
+
+	school = "transmutation"
+	charge_max = 600
```
Comment (ghost) [inline code/modules/spells/targeted/flash_to_coal.dm]: Isn't this a 6 seconds cooldown?

File: vgstation13.dme [hunk 5]
```diff
@@ -1788,6 +1788,7 @@
 #include "code\modules\spells\targeted\buttbots_revenge.dm"
 #include "code\modules\spells\targeted\disintegrate.dm"
 #include "code\modules\spells\targeted\ethereal_jaunt.dm"
+#include "code\modules\spells\targeted\flash_to_coal.dm"
```
Comment (ghost) [inline vgstation13.dme]: > flash_to_coal

b-but the spell is called flesh to coal

File: code/modules/spells/targeted/flash_to_coal.dm (hunk 3 above)
Comment (N3X15) [inline code/modules/spells/targeted/flash_to_coal.dm]: Which antag list?

File: code/modules/spells/targeted/flash_to_coal.dm (hunk 4 above)
Comment (N3X15) [inline code/modules/spells/targeted/flash_to_coal.dm]: yes

**Comment by Imagebard** [comment]
Did you just add a mask and exosuit to a human and called it a new race?

**Comment by ihadtoregisterforthis** [comment]
Definition of golem

**Comment by PJB3005** [comment]
Goddamnit @N3X15 you know better than to code shitcode like this...

WHY DID I ONLY HAVE ACCESS TO AN IPAD WHEN THIS WAS CREATED.

**Comment by Imagebard** [comment]
[WAKE ME UP INSIDE]

**Pull request merged**